tools/src/BUILD

```diff
@@ -51,7 +47,7 @@ go_library(
     srcs = glob(
         ["google/protobuf/*.pb.go"],
     ),
-    go_prefix = "//tools/src:go_prefix",
+    importpath = "github.com/cloudendpoints/esp",
```

Comment:
with this change, can we at least build it by
bazel build //tools/src/...

Otherwise, not need to change it

Reply:
Though with this change "bazel build //tools/src/..." still fails, I would like keep it because go_prefix is no longer valid syntax in new version of Bazel and we need to use importpath instead. Also this moves one more step toward making "bazel build //tools/src/..." successful.